wpilibNewCommands/src/main/java/module-info.java

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+module wpilib.commands2 {
```

Comment:
Open discussion: `wpilib.commands` or `wpilib.commands2`? The module names already differ from the package names.

Reply:
`commands2` makes sense if we plan on making a `commands3`, unless there's some way to version it separately. If we can use vendordep versioning and prevent users from installing two versions at once, we could just use `commands`.

Reply:
`commands2` makes the most sense. We'll have a commands3 in 2027, and each version is essentially a completely different library so reusing the same module name wouldn't be practical (or likely even possible)

Reply:
That argument doesn't make sense to me. WPILib 2008 and 2024 look completely different, yet we still call it wpilibj. Libraries going through a major API refactor are why major version numbers exist, so we should be using the major version for that instead of the library name itself.

Reply:
Because wpilib was changed gradually over time, without multiple incompatible versions concurrently being available for use. Commands v3 will probably be available side by side with v2 during a transition period

Reply:
That's the case for most libraries though, and they don't name their libraries like that. Giving them different names would allow users to use them both simultaneously, which we should explicitly disallow. Allowing that for v1 and v2 was a mistake.